core/data_migration_scripts/4_10_0_stages/EE_DMS_4_10_0_Event_Question_Group.dmsstage.php

```diff
@@ -40,17 +40,24 @@ protected function _migrate_old_row($event_question_group)
             global $wpdb;
             // If the question group was also for primary attendees, we should just update that row.
             // And we delete this row.
-            $success = $wpdb->update(
-                $this->_old_table,
-                ['EQG_additional' => true],  // data
-                [
-                    'EQG_primary' => true,
-                    'EVT_ID' => $event_question_group['EVT_ID'],
-                    'QSG_ID' => $event_question_group['QSG_ID']
-                ],  // where
-                array( '%d' ),   // data format
-                array( '%d', '%d', '%d' )  // where format
+            // Updating all the rows could be slow on massive DBs, so do the slow selection first, then a quick delete
+            // in order to avoid locking the table for too long.
```

Comment:
not that it matters *that* much but i think you meant to say "update" and not "delete"

ie:

```
// Updating all the rows could be slow on massive DBs, so do the slow selection first,
// then a quick update in order to avoid locking the table for too long.
```